V/VCDiff/build_tarballs.jl

```diff
@@ -0,0 +1,55 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "VCDiff"
+version = v"0.8.4"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/google/open-vcdiff.git", "868f459a8d815125c2457f8c74b12493853100f9")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/open-vcdiff/
+mkdir build
+git submodule update --init --recursive
+cd $prefix
+cmake -Dvcdiff_build_exec=OFF -DBUILD_SHARED_LIBS=ON $WORKSPACE/srcdir/open-vcdiff/
+make
+make install
+exit
```

Comment:
Please see the documentation: https://docs.binarybuilder.org/stable/build_tips/#CMake-builds
```suggestion
git submodule update --init --recursive
mkdir build && cd build
cmake .. \
    -DCMAKE_INSTALL_PREFIX=${prefix} \
    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
    -DCMAKE_BUILD_TYPE=Release \
    -Dvcdiff_build_exec=OFF \
    -DBUILD_SHARED_LIBS=ON
make -j${nproc}
make install
```